manas_cafa5/protein.py

```diff
@@ -44,6 +46,17 @@ def go_terms(self):
             self.load_uniprot()
         return self.terms.get('go')
 
+    def go_terms_children(self, graph):
+        return list(reduce(
+            lambda terms, term: terms.union(networkx.descendants(graph, term['id'])),
```

Comment:
In the issue description, I've talked about "direct descendants"... Those terms closer to the ones already assigned to proteins have more probability of being assigned during the 4 months that the competition lasts.

Let's use then the function [descendants_at_distance](https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.traversal.breadth_first_search.descendants_at_distance.html) and have `max_distance` as a parameter of this function.

Otherwise, since the graph is really huge, we will always have a very high amount of terms and the information won't be that useful. 

Sorry if I was not clear in the issue description.